src/components/analyser/details/BalanceSheetInfo.tsx

```diff
@@ -57,87 +58,161 @@ const useStyles = makeStyles(({ palette }: Theme) =>
   })
 );
 
+function checkValue(val: number): number {
+  let result = val;
+  if (val.toString() === 'None') {
+    result = 0;
+  }
+  return result;
+}
+
+function deleteName(val: number, text: string): string {
+  let result = text;
+  if (val === 0) {
+    result = '';
+  }
+  return result;
+}
+
 const BalanceSheetInfo: React.FC<BalanceSheetProps> = ({ companyReports }) => {
   const classes = useStyles();
   const { t } = useTranslation();
   const theme = useTheme();
 
+  const assetSeries = {
+    cashShortTermInvestments: checkValue(
+      companyReports.annualReports[0].cashAndShortTermInvestments
+    ),
+    inventory: checkValue(companyReports.annualReports[0].inventory),
+    receivables: checkValue(
+      companyReports.annualReports[0].currentNetReceivables
+    ),
+    physicalAsssets: checkValue(
+      companyReports.annualReports[0].propertyPlantEquipment
+    ),
+    deprecationAndAmortisation: checkValue(
+      companyReports.annualReports[0].accumulatedDepreciationAmortizationPPE
+    ),
+    intangibleAssets: checkValue(
+      companyReports.annualReports[0].intangibleAssets
+    ),
+    longTermInvestements: checkValue(
+      companyReports.annualReports[0].longTermInvestments
+    ),
+    otherCurrentAssets: checkValue(
+      companyReports.annualReports[0].otherCurrentAssets
+    ),
+    otherNonCurrentAssets: checkValue(
+      companyReports.annualReports[0].otherNonCurrrentAssets
+    ),
+  };
+
   const assets = [
     {
       data: [
         {
-          x: 'Cash & Short term Investments',
-          y: companyReports.annualReports[0].cashAndShortTermInvestments,
+          x: deleteName(
+            assetSeries.cashShortTermInvestments,
+            'Cash & Short term Investments'
```

Comment:
Muss alles übersetzt werden mit en.json in locales

Reply:
Klar kann ich die tage noch machen

Reply:
aber vllt unnötig weil fachbergriffe?

Reply:
Ha könnt ma Christoph fragen fällt aufjedenfall unter fix ;)